task.js

```diff
@@ -0,0 +1,67 @@
+const usersBirthdayYear = prompt('Уведіть Ваш рік народження')
+const currentYear = new Date(); 
+
+if (usersBirthdayYear !== null && usersBirthdayYear !== '') {
+    let date = new Date();  
+    let userYear = `Тобі ${date.getFullYear() - usersBirthdayYear}`; 
+
+    const usersCity = prompt ('В якому місті Ви живете?')
+
+    const capitals = ['Київ' , 'Вашингтон' , 'Лондон']
+    const countries = ['України' , 'США' , 'Великої Британії']
+    
+
+    if (usersCity !== null && usersCity !== ''){
+        if (usersCity === capitals[0]) {
+            let userCapitals = `Ти живеш у столиці ${countries[0]} `   // 'Ти живеш у столиці ' + countries[0];
+            alert( `${userYear} 
+${userCapitals}`)
+        } 
+        else if (usersCity === capitals[1]) {
+            let userCapitals = `Ти живеш у столиці ${countries[1]} `;
+            alert( `${userYear} 
+${userCapitals}`)
+        }
+         else if (usersCity === capitals[2]) {
+            let userCapitals = `Ти живеш у столиці ${countries[2]} `;
+            alert( `${userYear} 
```

Comment:
Good idea but this structure looks like key => value. So it will be better to create an object and work with it. 
`capitalsDataBase = {
 kyiv: 'Ukraine',
...
}`

and that you can get by answer. if answer Kyiv -> 

city = 'Kiyv'

message = `town is a capital of ${capitalsDataBase[city.toLowerCase()]}`